data/store/mongo/mongo_datum.go

```diff
@@ -601,7 +557,7 @@ func (d *DatumRepository) GetDataRange(ctx context.Context, userId string, typ s
 		return nil, errors.New("userId is empty")
 	}
 
-	if typ == "" {
+	if len(typ) < 1 {
```

Comment:
```suggestion
	if len(typ) == 0 {
```

Reply:
replaced all cases of len < 1 with == 0 throughout the file.